docs/src/DataStructures/ApplicationContext.md

```diff
@@ -0,0 +1,63 @@
+# Application Context
+
+## Schema
+
+An application context credential MUST be a W3C Verifiable Credential document containing information about the application requesting user sign-in.
```

Comment:
I'm not sure why this needs to be a W3C Verifiable Credential given that it is part of the signed payload.

Reply:
I guess it isn't, but the hash is, so still applies I think

Reply:
The fact that it is (or is not) signed by a particular issuer (read: certifier) may be of relevance to the wallet/dapp. Unsigned credentials are perfectly valid, but might come with a warning, similar to the browser experience when visiting a site that doesn't support HTTPS. The handling is left up to the browser (wallet).

Reply:
Looks like I was wrong about it being included in the payload of the signature. So it is correct that it needs to be signed.